src/hb-buffer-serialize.cc

```diff
@@ -91,26 +91,26 @@ hb_buffer_serialize_format_to_string (hb_buffer_serialize_format_t format)
 {
   switch ((unsigned) format)
   {
-    case HB_BUFFER_SERIALIZE_FORMAT_TEXT:	return serialize_formats[0];
-    case HB_BUFFER_SERIALIZE_FORMAT_JSON:	return serialize_formats[1];
+    case HB_BUFFER_SERIALIZE_FORMAT_TEXT: return serialize_formats[0];
+    case HB_BUFFER_SERIALIZE_FORMAT_JSON: return serialize_formats[1];
     default:
-    case HB_BUFFER_SERIALIZE_FORMAT_INVALID:	return nullptr;
+    case HB_BUFFER_SERIALIZE_FORMAT_INVALID:  return nullptr;
   }
 }
 
 static unsigned int
 _hb_buffer_serialize_glyphs_json (hb_buffer_t *buffer,
-				  unsigned int start,
-				  unsigned int end,
-				  char *buf,
-				  unsigned int buf_size,
-				  unsigned int *buf_consumed,
-				  hb_font_t *font,
-				  hb_buffer_serialize_flags_t flags)
+                                  unsigned int start,
+                                  unsigned int end,
+                                  char *buf,
+                                  unsigned int buf_size,
+                                  unsigned int *buf_consumed,
+                                  hb_font_t *font,
+                                  hb_buffer_serialize_flags_t flags)
```

Comment:
There is now a bunch of unrelated spacing changes. I have to admit, I never fully understood HarfBuzz indentation style and I probably mislead you.

Reply:
I'm not convinced that there is a consistent indentation style; that's why I converted everything to spaces (which is used *nearly* everywhere) in that file so there would be one.

![Screenshot 2020-09-18 at 14 28 45](https://user-images.githubusercontent.com/106728/93602948-43357980-f9bb-11ea-8786-22ced58d99a8.png)

Reply:
I believe the usual HB style (which TBH I've never liked, but that's beside the point) is a two-space unit of indentation, with real tabs set to a tab width of 8 and used where there are line-initial runs of 8+ spaces.

Reply:
Don't know whether editorconfig has a plugin in your editor or not but I use that to apply tab configuration on my editor, there is .editorconfig file in root of the project to guide the extension, not a fan of the style either.

Reply:
> I believe the usual HB style (which TBH I've never liked, but that's beside the point) is a two-space unit of indentation, with real tabs set to a tab width of 8 and used where there are line-initial runs of 8+ spaces.

Right. And I inherited this from GNOME / FreeDesktop projects I was acquainted with. Open to alternative proposals.

Reply:
I use the `.editorconfig`, but it indents using tabs:

```
indent_style = tab # should be space
```

Reply:
See: https://github.com/editorconfig/editorconfig-vscode/issues/190